api/src/services/validation-service.ts

```diff
@@ -397,11 +400,13 @@ export class ValidationService extends DBService {
   async persistValidationResults(csvState: ICsvState[], mediaState: IMediaState): Promise<boolean> {
     defaultLog.debug({ label: 'persistValidationResults', message: 'validationResults' });
 
-    let parseError = false;
+    const parseError = false;
```

Comment:
Lets remove this `parseError` entirely, and the return at the end of the function. We can just make this function a Promise Void type.

We only ever use the original return value in 1 spot, and its for a test. We can just update that test to assert that if no exception is thrown, it passes, since that is really what this function does now.

Reply:
resolved in both validate-service and summary-service